website/blog/2023-02-02-5-database-alternatives-mongodb-2023.md

```diff
@@ -0,0 +1,121 @@
+---
+slug: 5-database-alternatives-mongodb-2023
+title: "MongoDB Alternatives: 5 Database Alternatives to MongoDB for 2023"
+author: Alexander Fashakin
+image: /img/blog/mongodb-alternatives.png
+description: "The top 5 MongoDB-compatible alternatives to MongoDB include: FerretDB, DocumentDB, CosmosDB, GaussDB(for Mongo), and ToroDB."
```

Comment:
Any reason why those 5? Why, for example, we talk about abandoned ToroDB and not https://mariadb.com/kb/en/mariadb-maxscale-6-nosql-protocol-module/ ?

Reply:
I was focusing on MongoDB compatible databases. These ones were built mostly as direct alternatives... The inclusion of ToroDB is to have another open source db in it besides FerretDB (even though its abandoned).

Reply:
We can have other listicles where we include this one from MariaDB as well

Reply:
That link is not about MariaDB database. In fact, their product (MaxScale NoSQL Protocol Module) is the closest to us, I think

Reply:
Interesting! Thanks for mentioning this, if you come across other similar tools too, please let me know.

I'm almost inclined to add it to the article, but this should be published today. We can do more listicles and have MaxScale in those ones.

Reply:
@AlekSi I just updated it to include MariaDB Maxscale nosql protocol. PTAL